platform/kubernetes/wrapper/wrap.go

```diff
@@ -0,0 +1,85 @@
+package wrapper
+
+import (
+	"errors"
+	"io/ioutil"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+	"strings"
+
+	"github.com/sipb/homeworld/platform/keysystem/worldconfig/paths"
+)
+
+func GetConf(filename string) (map[string]string, error) {
+	conf, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	kvs := map[string]string{}
+	for _, line := range strings.Split(string(conf), "\n") {
+		line = strings.TrimSpace(line)
+		if len(line) > 0 && line[0] != '#' {
+			kv := strings.SplitN(line, "=", 2)
+			if len(kv) != 2 {
+				return nil, errors.New("incorrectly formatted cluster.conf")
```

Comment:
This error message should indicate the actual filename instead of assuming it was cluster.conf

Also - where do these configuration files come from and why this configuration format?

Reply:
These configuration files are generated by a combination of spire and the keysystem. They're in this format so that they can be source'd by bash scripts, though this commit does eliminate most of those uses.

They should probably be converted to YAML at some point, once we finish eliminating bash launch scripts. I've filed #431 to remind us of this.